src/components/Dialogs/Dialog/Dialog.tsx

```diff
@@ -18,42 +21,66 @@ export interface DialogProps extends ModalProps {
     /** Dialog title */
     title?: string;
     /** Properties of the accept button */
-    acceptButton: ButtonProps;
+    acceptButton?: ButtonProps;
     /** Properties of the cancel button */
     cancelButton?: ButtonProps;
+    /** the context of the buttons in the footer */
+    variant?: Omit<ButtonContext, 'secondary'>;
+    /** closes the dialog */
+    onClose: () => void;
 }
 
 const { block, elem } = bem('Dialog', styles);
 
 export const Dialog: React.FC<DialogProps> = (props) => {
-    const { isOpen, children, title, acceptButton, cancelButton, ...rest } = props;
+    const {
+        isOpen,
+        children,
+        title,
+        acceptButton,
+        cancelButton,
+        variant = 'primary',
+        onClose,
+        ...rest
+    } = props;
 
-    const isConfirm = !!cancelButton;
+    const { ref, height } = useResizeObserver<HTMLDivElement>();
+    const showButtons = !!acceptButton;
 
     return (
         <Modal {...rest} {...block(props)} isOpen={isOpen}>
-            <div {...elem('content', props)} role="alert">
-                {!!title && (
+            {!!title && (
+                <div {...elem('header')} role="alert">
                     <Heading level="h2" {...elem('title', props)}>
                         {title}
                     </Heading>
-                )}
+                    <button {...elem('closeButton')} type="button" onClick={onClose}>
+                        <MdClose />
+                    </button>
+                </div>
+            )}
+            <div ref={ref} {...elem('content')}>
                 {children}
             </div>
-            <div {...elem('actions', props)}>
-                {isConfirm && (
+            {showButtons && (
```

Comment:
there is no padding between the buttons

Reply:
This is not resolved. Looking at the vercel deployment, there is still no padding between the buttons.

Reply:
I had padding between the buttons before the last push. I had added a class on the cancel button and add padding. However, after the first comment I thought that there shouldn't be padding, so I removed it. Should I add padding by another way or what is needed here?